sycl/doc/GetStartedGuide.md

```diff
@@ -36,6 +36,9 @@ Throughout this document `DPCPP_HOME` denotes the path to the local directory
 created as DPC++ workspace. It might be useful to create an environment variable
 with the same name.
 
+The most up-to-date source for configuration values is [buildbot/configure.py](../../buildbot/configure.py).
+If you run into configuration problems you can validate with that file.
```

Comment:
Thanks for updating the documentation.
What do you think if we replace `cmake` run commands below with the run command using `configure` script? This way, cmake options changes will not require documentation updates.

Reply:
yes sure, i can also document that instead.

So the docs should end up pointing to run the configure script?
shall I include args in the docs?

how about projects including llvm as a subdirectory?

Reply:
> yes sure, i can also document that instead.
> 

Thanks!

> So the docs should end up only pointing out to run the configure script?

I suggest replacing two cmake commands you've changed in this PR. The rest of the commands seems to be more stable.

> shall I include args in the docs?

I think it makes sense to provide args, which will run the cmake command. E.g. second command should include `--cuda` argument.

> 
> how about projects including llvm/sycl as a cmake subdirectory?

This use case is not covered by these docs and I assume if SYCL project is included as a subproject to another project X, the project X should define how SYCL sub-project is configured.
Anyway, we can use the hint similar to what you added here i.e. by default, please, use configure script, if it's not enough for your needs, you can use CMake directly.

Reply:
Ok, sure. i'll look into it and update this PR.

So, after all just the short `python configure.py` with the most important args and the hint to look in the configure.py for any additional configuration values.